Dockerfile

```diff
@@ -1,15 +1,15 @@
 # Build container
 FROM node:6-alpine as build_container
 
-WORKDIR /home/node
+WORKDIR /root
 
 RUN apk add --no-cache git python g++ make bash libc6-compat
 
-USER node
+USER root
```

Comment:
Keep using `node` as user, in Dokerfile you can make all your build with users privilege.